libwayshot/src/lib.rs

```diff
@@ -492,7 +486,7 @@ impl WayshotConnection {
                 }
 
                 surface.set_buffer_transform(output_info.transform);
-                surface.set_buffer_scale(output_info.scale);
+                // surface.set_buffer_scale(output_info.scale());
```

Comment:
why comment this line? will this line never use again?

Reply:
This accepts only an integer and if we were to round it, it'd still be incorrect. It needs fractional scaling support.